ParisBNF/abb/BNFabb47.xml

```diff
@@ -0,0 +1,313 @@
+                              <textLang mainLang="gez"/>
+                           </msItem>
+
+
+
+                           <msItem xml:id="ms_i1.1.3">
+                              <title type="complete" ref="LIT2705Introd#Exaltation" cert="medium">The Exaltation of the Gospels</title>
+                           </msItem>
+
+                           <msItem xml:id="ms_i1.1.4">
+                             <locus from="11r"/>
+                              <title type="incomplete" ref="LIT1224Canons">ሰንጠረጅ</title>
+                              <textLang mainLang="gez"/>
+                           </msItem>
+                        </msItem>
+
+                      </msItem>
+
+                              <msItem xml:id="ms_i1.2">
+                           <locus from="18r"/>
+                           <title type="complete" ref="LIT1558Matthew"/>
+                           <textLang mainLang="gez"/>
+
+                           <msItem xml:id="ms_i1.2.1">
+                              <locus from="18r"/>
+                              <title type="complete" ref="LIT2709Matthew"></title>
+                              <textLang mainLang="gez"/>
+                              </msItem>
+
+                              <msItem xml:id="ms_i1.2.2">
+                                  <locus from="66v"/>
+                                 <title type="complete" ref="LIT2709Matthew#BiographyMatthew" cert="medium"></title>
+                                 <textLang mainLang="gez"/>
+                                 </msItem>
+                           </msItem>
+
+                           <msItem xml:id="ms_i1.2.3">
+                               <locus from="67v"/>
+                              <title type="complete" ref="LIT2709Matthew#TituliMatthew"></title>
+                              <textLang mainLang="gez"/>
+                              </msItem>
+
+
+
+                           <msItem xml:id="ms_i1.3">
+                              <locus from="68r"/>
+                              <title type="complete" ref="LIT1882MarkGo"/>
+                              <textLang mainLang="gez"/>
+
+                              <msItem xml:id="ms_i1.3.1">
+                                <locus from="68r"/>
+                                 <title type="complete" ref="LIT2711Mark"></title>
+                                 <textLang mainLang="gez"/>
+                              </msItem>
+
+                              <msItem xml:id="ms_i1.3.2">
+                                  <locus from="95r"/>
+                                 <title type="complete" ref="LIT1882MarkGo#BiographyMark"/>
+                                 <textLang mainLang="gez"/>
+                              </msItem>
+
+                              <msItem xml:id="ms_i1.3.3">
+                                <title type="complete" ref="LIT1882MarkGo#TituliMark" cert="low"/>
```

Comment:
no folio given?

Reply:
Not at all sure that the text is there. The description of Abbadie 47 in the catalogue fragmentary, the manuscript is not online.